ref(astro): Improve Getting Started introduction (#8685)
getsentry/sentry-docs: src/platform-includes/getting-started-primer/javascript.astro.mdx

</Alert>

## Prerequisites

Before we get started, make sure you have the following:

> - A Sentry account and a project.

Reviewer: ```suggestion
- A Sentry account.
```

  - Don't have an account yet? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.

> - An Astro project using Astro `3.0.0` or newer.

Reviewer: ```suggestion
- An Astro project that uses Astro `3.0.0` or newer.
```

> - This SDK currently only works on Node runtimes.
>   - Non-Node runtimes, like Vercel's Edge runtime or Cloudflare Pages, are currently not supported.

Reviewer: ```suggestion
- Node runtimes.
  - This SDK currently only works on Node runtimes. Non-Node runtimes, like Vercel's Edge runtime or Cloudflare Pages, are currently not supported.
```
